Snippets/Core/Core_6/Msmq/DLQOptInForTTBR.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Core6.Msmq
+{
+    using NServiceBus;
+
+    class DLQOptInForTTBR
+    {
+        DLQOptInForTTBR(EndpointConfiguration endpointConfiguration)
+        {
+            #region msmq-dlq-for-ttbr-optin
+
+            endpointConfiguration.UseTransport<MsmqTransport>()
+                .UseDeadLetterQueueForMessagesWithTimeToReachQueue();
```

Comment:
For example: 
https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Transports/Msmq/MsmqMessageDispatcher.cs#L65-L67
This is all about not DLQing based on TTBR not TTRQ, yet the API is called TTRQ. IMO, that's incorrect. 
@SimonCropp - thoughts on this?